### ISSUE TITLE
Import problem with openxlsx

### ISSUE THREAD

**Issue by mrustl** [KWB-R/kwb.budget#3]
Maybe an openxlsx issue (in https://github.com/KWB-R/kwb.budget/commit/2be6cc2bc2524c0af6f713ef86056a63f37b5455). But as old version with kwb.db is not working on my computer I don`t know wether this also happens there @hsonne.

 ```
file <- "...../xxx_Arctik_FINAL.xlsx"
region_names <- as.character(openxlsx::getNamedRegions(file))

  lapply(region_names, function(name) {
    try(openxlsx::read.xlsx(file, namedRegion = name))
  })

```

```
print(region_names)
 [1] "average_person_month"       "details_consumables"        "details_equipment"          "details_personnel"         
 [5] "details_subcontracting"     "details_travelling"         "range_average_person_month" "range_consumables"         
 [9] "range_contact"              "range_direct"               "range_equipment"            "range_indirect"            
[13] "range_partner"              "range_personnel"            "range_subcontracting"       "range_total"               
[17] "range_travelling"           "selectwp"                   "threshold_justification"    "total_consumables"         
[21] "total_direct"               "total_equipment"            "total_indirect"             "total_personnel"           
[25] "total_subcontracting"       "total_travelling" 

[[18]]
[1] "Error in gsub(sheet, \"\", region, fixed = TRUE) : \n  ungültiges 'pattern' Argument\n"
attr(,"class")
[1] "try-error"
attr(,"condition")
<simpleError in gsub(sheet, "", region, fixed = TRUE): ungültiges 'pattern' Argument>
```

**Comment by hsonne** [comment]
This is the only range. I think we do not need it. Looks like an error (the only range name without underscore). I now issue a warning about ranges that could not be read.

**Comment by hsonne** [comment]
Look at this! The problematic range points to the non-existing referenced file:

![grafik](https://user-images.githubusercontent.com/11964315/83270788-41a43380-a1c9-11ea-8f70-d06f57eed034.png)

**Comment by hsonne** [comment]
No need for the existing files, I think. But important for the new template. I sent you a new version of the template via e-mail. I needed to save the file as `.xls` in order to be able to delete the named range in the names manager. In the original `.xlsx` file (in any `.xlsx` file?) the buttons to add/modify/delete ranges are greyed out. I deleted the range in the `.xls` file and saved back to `.xlsx`.
